packages/docs/content/components/date-field/single-input-date-field.mdx

```diff
@@ -26,6 +26,10 @@ medicare:
 
 <SeeStorybookForGuidance storyId={'components-singleinputdatefield--docs'} />
 
+### Web Component
```

Comment:
Should we add a note here to indicate that, although the `ds-date-field` web component and the `SingleInputDateField` React component have different names, they serve as counterparts in functionality?